### ISSUE TITLE
Add support for Kafka client metrics in device registry

### ISSUE THREAD

**Issue by calohmn** [eclipse/hono#3158]
Configuring the Quarkus based Device Registry with a Kafka client metricsPrefixes configuration
```
hono:
  kafka:
    metrics:
      metricsPrefixes:
        - kafka.consumer.fetch.manager.bytes.consumed.total
```
causes startup of the component to fail:
```
Mar 16, 2022 9:20:51 AM io.quarkus.runtime.ApplicationLifecycleManager run
ERROR: Failed to start application (with profile prod)
io.smallrye.config.ConfigValidationException: Configuration validation failed:
	hono.kafka.metrics.metricsPrefixes does not map to any root
	hono.kafka.metrics.metricsPrefixes[0] does not map to any root
	at io.smallrye.config.ConfigMappingProvider.mapConfiguration(ConfigMappingProvider.java:875)
	at io.smallrye.config.ConfigMappingProvider.mapConfiguration(ConfigMappingProvider.java:831)
	at io.smallrye.config.SmallRyeConfigBuilder.build(SmallRyeConfigBuilder.java:420)
	at io.quarkus.runtime.generated.Config.readConfig(Unknown Source)
	at io.quarkus.deployment.steps.RuntimeConfigSetup.deploy(Unknown Source)
	at io.quarkus.runner.ApplicationImpl.doStart(Unknown Source)
	at io.quarkus.runtime.Application.start(Application.java:101)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:104)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:67)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:41)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:120)
	at io.quarkus.runner.GeneratedMain.main(Unknown Source)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:53)
```

**Comment by calohmn** [comment]
Kafka client metrics support hasn't been added yet to the device registry implementations (it hasn't been done in #2848).

**Comment by calohmn** [comment]
Removing the `bug` label - in released Hono versions, the Kafka client metrics are documented to be supported only in protocol adapters and the Command Router.

**Comment by calohmn** [comment]
Fixed via e57a7b39dbab91463f65bb70b5874e7944a05933.
